.github/workflows/update-gradle-wrapper.yml

```diff
@@ -0,0 +1,17 @@
+name: Update Gradle Wrapper
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
When the hole world run there jobs at midnight we have a big energy consumption peak.
What's about to run it on a random other time and is it necessary every day ?

I know these are just mini-minimal effects, but we should start saving energy on every task we see !

Reply:
That's a good call @hannesa2! Even running this action weekly is good enough. Which value do you want me to set for the schedule?

Reply:
I'm fine with a weekly action

Reply:
I've now updated the schedule to run the action every monday at 6am.